core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -45,13 +45,20 @@ namespace Kokkos {
 namespace Impl {
 
 inline bool execute_in_serial(OpenMP const& space = OpenMP()) {
+  // FIXME_OPENMP - `omp_get_max_active_levels` fails with gcc version lower
+  // than 11.2.0
+#if KOKKOS_COMPILER_GNU >= 1120
   return (OpenMP::in_parallel(space) && !(
 #if _OPENMP >= 201511
                                             (omp_get_max_active_levels() > 1)
 #else
                                             omp_get_nested()
 #endif
                                             && (omp_get_level() == 1)));
+#else
+  return (OpenMP::in_parallel(space) &&
+          !(omp_get_nested() && (omp_get_level() == 1)));
+#endif
```

Comment:
Maybe
```suggestion
  // FIXME_OPENMP - `omp_get_max_active_levels` fails with gcc version lower
  // than 11.2.0
#if KOKKOS_COMPILER_GNU >= 1120 && _OPENMP >= 201511
  int is_nested = omp_get_max_active_levels() > 1;
#else
  int is_nested = omp_get_nested();
 #endif
  return (OpenMP::in_parallel(space) &&
          !(is_nested && (omp_get_level() == 1)));
```
to keep it a little more compact?

Reply:
Done. Thanks for the suggestion.